simulators/nav/src/utils/utils.ts

```diff
@@ -8,7 +10,9 @@ import {
   OdomFormat,
   Point2D,
   Speeds,
-  ZedGimbalPosition
+  ZedGimbalPosition,
+  ObstacleNewInterfaceMessage,
+  ObstacleMessage
```

Comment:
why are there two of these? is one of them the old one? and if so, why are we keeping it?

Reply:
Because I did not want to mess up with how the current 2d sim detects obstacles within the FOV, I basically did a conversion of the old obstacle message (distance, bearing) to its bounding box using the size of the obstacle.

Reply:
It would be better to rename the old obstacle message. Will do that later

Reply:
Never mind, I can get rid of the old one entirely after making some changes